junit-jupiter-api/src/main/java/org/junit/jupiter/api/AssertLinesMatch.java

```diff
@@ -173,7 +171,9 @@ void assertLinesMatchWithFastForward() {
 					}
 				}
 
-				fail("expected line #%d:`%s` doesn't match", expectedLineNumber, snippet(expectedLine));
+				int actualLineNumber = actualLines.size() - actualDeque.size() + 1; // 1-based line number
+				fail("expected line #%d doesn't match actual line #%d%n" + "\texpected: `%s`%n" + "\t  actual: `%s`",
```

Comment:
```suggestion
				fail("expected line #%d doesn't match actual line #%d%n\texpected: `%s`%n\t  actual: `%s`",
```

Reply:
Ah... my fault. I wanted to start each line in an extra line. Will add `//` to force line breaks.